common/quickstart/aws-docker-helper.sh

```diff
@@ -0,0 +1,31 @@
+# Helper for the AWS specific host, to run a species/dataset instance of the Knetminer Docker container with AWS logging.
+#
+# See https://github.com/Rothamsted/knetminer/wiki/8.-Docker for details. See https://github.com/Rothamsted/knetminer/wiki/3.-Deploying-KnetMiner-with-Docker#deploying-with-aws-analytics-enabled for additional help.
+
+# Set the default directory
+cd "$(dirname "$0")"
+# Get the initial arguments & make sure it's the aws flag - More flags can be added here
+aws_flag="$1"; 
+aws_dir="$2"; shift;
+
+if [ "$aws_flag" != "--aws" ]; then echo -e "\nIncorrect flag given, you must give the --aws flag first!\n\nExiting script\n"; exit 1; fi
+
+# Make sure the directory and files exist!
+if [ -d "$aws_dir" ]; then
+	if [ -f "$aws_dir/.aws/credentials" ] && [ -f "$aws_dir/analytics-s3-sync.sh" ]; then
+		DOCKER_OPTS="$DOCKER_OPTS -v $aws_dir/.aws/credentials:/root/.aws/credentials:ro" 
```

Comment:
will add `-it`, which is usually a useful default (and overridden if your pass your options).